src/types/zetkin.ts

```diff
@@ -171,3 +171,35 @@ export interface ZetkinActivity {
     title: string;
     info_text: string | null;
 }
+
+export enum TaskType {
+    demographic = 'demographic',
+    offline = 'offline',
+    share_link = 'share_link',
+    share_image = 'share_image',
+    visit_link = 'visit_link',
+    watch_video = 'watch_video',
+}
+
+export interface ZetkinTask {
+    id: number;
+    title: string;
+    instructions: string;
+    published: string; // iso string
+    expires: string; // iso string
+    deadline: string; // iso string
+    type: TaskType;
+    config: Record<string, unknown >;
+    target: {
+        filter_spec: Array<{type: string}>;
```

Comment:
I'm not exactly sure if this type is correct

Reply:
The type of a filter is:
```typescript
{
  type: string;
  op: 'add' | 'sub';
  config?: Record<string, unknown>;
}
```

Full documentation here: https://github.com/zetkin/zetkin-platform/tree/master/services/ps-person-query-service

Reply:
Cheers